docs/src/create-apps/runtime-operations.md

```diff
@@ -0,0 +1,333 @@
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "next-rebuild", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+{{< /codetabs >}}
+
+### Execute actions on your Node.js app
+
+You can define runtime operations for common [pm2](https://pm2.io/docs/runtime/overview/) process manager tasks. 
+
+{{< codetabs >}}
++++
+title=Ping your app
++++
+
+To ping your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-ping:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 ping $APP
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-ping", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Reload your app
++++
+
+To reload your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-reload:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 reload $APP
+``` 
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-reload", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Restart your app
++++
+
+To restart your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-restart:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 restart $APP 
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-restart", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+{{< /codetabs >}}
+
+### Define management commands on your Django project
+
+On a Django project, you can [define custom `django-admin` commands](https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/), for example to run a migration outside of the Django ORM.
```

Comment:
Instead of `for example to run a migration outside of the Django ORM.` I would say something like:
`for example to run a one-off management command outside of the Django ORM migration framework`